applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -69,18 +69,18 @@ def GetDefaultParameters(cls):
                 "dynamic_tau": 1.0,
                 "surface_tension": false
             },
-            "bfecc_convection" : false,
-            "bfecc_number_substeps" : 10,
+            "levelset_variable_name" : "DISTANCE",
+            "levelset_convection_variable_name" : "VELOCITY",
+            "levelset_gradient_variable_name" : "DISTANCE_GRADIENT",
```

Comment:
These are better to be encapsulated within "levelset_convection_settings".

Reply:
I don't think so. See my comment above.
1) These are not only used by the levelset convection but for the gradient, smoothing, etc.
2) And more important, encapsulating them impedes using the standard defaults mechanism to give value to them. If these are encapsulated we need to either recursively validate the user settings, meaning that we can no longer use the mechanism in the base Python solver, or to do a manual validation in the constructor.

In any case, what I'd do is to remove these from the input and hardcode them to `DISTANCE`, `VELOCITY` and `DISTANCE_GRACIENT` in the constructor, specially taking into account that the NS element assumes distance and velocity which is to say that "custom" variables, as we do in the conv-diff solvers, is not really supported.

Reply:
I agree with "hardcode them to DISTANCE, VELOCITY and DISTANCE_GRADIENT" inside the python solver (not revealed to the user in the settings). This makes sense.

Reply:
Done!